I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 52, in script
    $ not_imposter = not_imposter +1 #Good Answer!
  File "game/script.rpy", line 52, in <module>
    $ not_imposter = not_imposter +1 #Good Answer!
NameError: name 'not_imposter' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 52, in script
    $ not_imposter = not_imposter +1 #Good Answer!
  File "renpy/ast.py", line 928, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "renpy/python.py", line 2245, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/script.rpy", line 52, in <module>
    $ not_imposter = not_imposter +1 #Good Answer!
NameError: name 'not_imposter' is not defined

Windows-10-10.0.19041
Ren'Py 7.4.10.2178
MothGame 1.0
Tue May 30 11:19:43 2023
